### DISCUSSION TITLE
TouchEffect doesnt work in UWP? How to use LongPressCommandParameter?

### DISCUSSION THREAD

**Discussion by TuxCZ** [xamarin/XamarinCommunityToolkit#1454] [Q&A]
Hello,
I have problem with a touch effect. It works perfectly in the Android APP, but in UWP doesnt. 

My UWP target version is Windows 10, 2004 (Build 19041) and Min version 1803, build 17134...

I tried this fix:
```
 var assembliesToInclude = new List<Assembly>()
                {
                    typeof(Xamarin.CommunityToolkit.Effects.TouchEffect).GetTypeInfo().Assembly
                };
                Xamarin.Forms.Forms.Init(e, assembliesToInclude);
```

but it doesnt helped me. I changed ,,Compile with .NET Native tool chain" but it doesnt helped too.. I tried to compile UWP with different target and min versions, but it doesnt work...

My second question is how to use LongPressCommandParameter? I have entry defined like this:
` <Entry x:Name="txtProjectName" Text="{Binding Project.Name}" Style="{StaticResource EntrySecondaryStyle}"
                               xct:TouchEffect.LongPressCommand="{Binding LongPressCommand}"
                               xct:TouchEffect.LongPressCommandParameter="{Binding .}"
                               xct:TouchEffect.LongPressDuration="1500"/>`

And I need to get LongPressCommandParameter (sender) in the code... How can I do that?

**Comment by AndreiMisiukevich** [comment]
@TuxCZ what XCT version do you use?

Can you try the latest "pre-release" version?

As for another question
```xct:TouchEffect.LongPressCommandParameter="{x:Reference txtProjectName}"```

**Reply by TuxCZ** [reply]
I use 1.1.0 version...  I tried prelease version, but It didnt helped...

**Reply by AndreiMisiukevich** [reply]
Also, could you please check if regular Command works fine (on BoxView for example).

Try to set Min and Target versions as high as possible and test them too.

**Reply by TuxCZ** [reply]
I tried it on the boxView but not working...
I set both versions as high as possible and still not working...

**Reply by AndreiMisiukevich** [reply]
Have you tried the latest pre-release?

**Reply by TuxCZ** [reply]
Yes, I tried latest pre-release... nothing changed
